assets/css/uswds-theme.scss

```diff
@@ -183,3 +183,14 @@ $theme-sidenav-breakpoint: '{{ site.data.navigation.sidenav-breakpoint | default
 
 // import user custom styles
 @import 'uswds-theme-custom-styles';
+
+// table style overrides
+main .usa-table td {
+  vertical-align: top;
+  font-size: 1.1rem;
+}
+main .usa-table td ul,
+main .usa-table td li,
+main .usa-table td strong {
+  font-size: inherit;
+}
```

Comment:
Thanks @maggiewachs. To be a stickler, could you move the CSS to the custom styles file https://github.com/CivicActions/accessibility/blob/main/_sass/_uswds-theme-custom-styles.scss?

Reply:
Np!